package.json

```diff
@@ -80,7 +81,7 @@
   "eslintConfig": {
     "parser": "@typescript-eslint/parser",
     "parserOptions": {
-      "ecmaVersion": "2018",
+      "ecmaVersion": 2018,
```

Comment:
Got this warning in VSCode:

```
Value is not accepted. Valid values: 3, 5, 6, 2015, 7, 2016, 8, 2017, 9, 2018, 10, 2019, 11, 2020, 12, 2021.(1)
```

Reply:
Not sure why eslint itself didn't complain, but :+1: